### PR TITLE
Some minor additions

### PR DESCRIPTION
**Pull request by terrytangyuan**

### CODE REVIEW

**Comment by terrytangyuan** [comment]
This time passed my local tests.

**Comment by terrytangyuan** [comment]
I'll let you know when I finish. Thanks.

File: setup.py [hunk 1]
```diff
@@ -21,5 +24,8 @@
           'scipy',
           'scikit-learn'
       ],
-      test_suite='test'
+      test_suite='test',
+      keywords=['Metric Learning', 'Large Margin Nearest Neighbor', \
```
Comment (perimosocordiae) [inline setup.py]: The line continuations aren't necessary here. I would format this like the list of `install_requires`, with one per line.

**Comment by terrytangyuan** [comment]
@perimosocordiae thanks. Fixed. Do you mind change all the names from `metric_learn` to `metric-learn`, which is easier to recognize. After that I am ready to submit the initial version. I've set up all the necessary environment.

**Comment by terrytangyuan** [comment]
I meant change the names after this merge plz.

**Comment by perimosocordiae** [comment]
Sounds good, merged. I don't much care if it's metric_learn or metric-learn, so feel free to change that if you like.

**Comment by terrytangyuan** [comment]
@perimosocordiae Travis build failed since you merged while it's still running. I'll submit a new PR to fix that or you can add me to admin if possible. I cannot change the package name or repo name if i am not admin.

**Comment by perimosocordiae** [comment]
That's fine. I'll take care of the repository name.

**Comment by terrytangyuan** [comment]
Okay great! Can you change it to point to the new name for Travis as well? You might need to activate the new repo or something.

**Comment by perimosocordiae** [comment]
Done, just needed to tell Travis to re-sync.

**Pull request merged**